### DISCUSSION TITLE
Laravel reset password

### DISCUSSION THREAD

**Discussion by AhmedFathyEmam** [postalserver/postal#2077] [Development discussions]
"When I tried to add with Gmail, everything worked, but when I switched to mail.getfoza.com, this error happened.
Laravel version 9
Php 8
![image](https://user-images.githubusercontent.com/105523131/179384199-8c35e30b-9d66-4ae1-9905-9a99afcde0c2.png)
Connection could not be established with host "mail.getfozo.com:2525": stream_socket_client(): Unable to connect to mail.getfozo.com:2525 (A socket operation was attempted to an unreachable network)

**Comment by willpower232** [comment]
You should check with `postal status` that your Postal is running and on what port with `netstat -lnp`.

If Postal is definitely available on that port, you should double check any firewalls.
